### PR TITLE
Speed up tests/api/ from 20s down to 6s

### PR DESCRIPTION
**Pull request by ashb**
Not a big over-all speed up, but also not a big change

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: pylintrc-tests [hunk 1]
```diff
@@ -389,7 +389,7 @@ min-similarity-lines=4
 # Regular expression matching correct argument names. Overrides argument-
 # naming-style.
 # BasPH: Kept snake_case but allow for 1-char argument names.
-argument-rgx=[a-z_][a-z0-9_]{0,30}$
+argument-rgx=[a-z_][a-z0-9_]{0,32}$
```
Comment (ashb) [inline pylintrc-tests:392]: `minimal_app_for_experimental_api` was longer than 30 characters.

File: tests/api/common/experimental/test_mark_tasks.py [hunk 2]
```diff
@@ -68,22 +63,19 @@ def setUp(self):
...
         for dr in drs:
             dr.dag = self.dag2
-            dr.verify_integrity()
 
         drs = _create_dagruns(
             self.dag3, self.dag3_execution_dates, state=State.SUCCESS, run_type=DagRunType.MANUAL
         )
         for dr in drs:
             dr.dag = self.dag3
-            dr.verify_integrity()
```
Comment (ashb) [inline tests/api/common/experimental/test_mark_tasks.py:86]: Called when creating the dag run, we don't need to do again.

File: tests/api/common/experimental/test_mark_tasks.py [hunk 3]
```diff
@@ -568,8 +560,6 @@ def test_set_failed_dag_run_to_running(self):
         middle_time = timezone.utcnow()
         self._set_default_task_instance_states(dr)
 
-        time.sleep(2)
```
Comment (ashb) [inline tests/api/common/experimental/test_mark_tasks.py:571]: Doesn't seem  to have any effect.

**Review by kaxil — APPROVED** [review]

**Comment by potiuk** [comment]
+1

**Pull request merged**